open_feature/evaluation_context/evaluation_context.py

```diff
@@ -0,0 +1,12 @@
+class EvaluationContext:
+    def __init__(self, targeting_key: str = None, attributes: dict = None):
+        self.targeting_key = targeting_key
+        self.attributes = attributes or {}
+
+    def merge(
+        self, ctx1: "EvaluationContext", ctx2: "EvaluationContext"
```

Comment:
Isn't `self` ctx1 in this case? Or should this be a static method? It appears that here, we're working with 3 different EvaluationContext objects which seems wrong?

Reply:
it should be static

Reply:
Yes agreed, self is the first context, I've made that change now